std/container/array.d

```diff
@@ -1282,6 +1286,13 @@ if (!is(immutable T == immutable bool))
     }
 }
 
+@system unittest
+{
+    import std.algorithm : equal;
```

Comment:
it seems that the style checker wants you to be more specific with the import (e.g. `import std.algorithm.comparison`)

Reply:
@RazvanN7 sorry for the delay, now fixed.